stdlib/3.4/asyncio/streams.pyi

```diff
@@ -95,12 +95,12 @@ class StreamReader:
     def set_transport(self, transport: transports.BaseTransport) -> None: ...
     def feed_eof(self) -> None: ...
     def at_eof(self) -> bool: ...
-    def feed_data(self, data: bytes): ...
+    def feed_data(self, data: bytes) -> None: ...
     @coroutines.coroutine
     def readline(self) -> Generator[Any, None, bytes]: ...
     @coroutines.coroutine
-    def readuntil(self, separator=b'\n') -> Generator[Any, None, bytes]: ...
+    def readuntil(self, separator: bytes = b'\n') -> Generator[Any, None, bytes]: ...
```

Comment:
While you're at it, can you change these defaults to `...`? In stubs it doesn't matter what the default value is, just that there is one.